CHANGELOG.md

```diff
@@ -17,3 +26,5 @@ All notable changes to this project will be documented in this file.
 ## 0.1.0 - 2020-12-03
 ### Added
 - Initial release.
+
+[parity_scale_codec]: https://docs.rs/parity-scale-codec
```

Comment:
CHANGELOG can be large in size, I think the url should be inline.

Reply:
fixed